ngraph_bridge/ngraph_utils.cc

```diff
@@ -350,6 +350,84 @@ Status CheckAxisDimInRange(std::vector<int64> axes, size_t rank) {
   return Status::OK();
 }
 
+#if (CMAKE_BUILD_TYPE == Debug)
+// GDB Debug utility
+void gdb_print_ngfunc_nodes_shptr(const shared_ptr<ngraph::Function>& func) {
+  gdb_print_ngfunc_nodes_funcref(*func);
+}
+
+void gdb_print_ngfunc_nodes_vptr(void* func_addr) {
```

Comment:
Not needed. This is something you can do in a debugger.

Reply:
I son't know how. Any example?

Reply:
If it only needs a cast, you can do that in gdb.